### DISCUSSION TITLE
GitIgnore

### DISCUSSION THREAD

**Discussion by ElectronicsArchiver** [Rnen/AdminToolbox#128] [General]
Could you please sort out what the 
repo actually uses from `.gitignore`.

Most is irrelevant and like just now,
prevented me from commiting some files.

> [Ll]og/

Which I don't think is relevant to the repo?

*I'd do it myself, though I don't know what C# projects use~*

**Comment by Rnen** [comment]
The Repo uses a standard VS Studio C# project .gitignore. Everything except the following section is standard practice.
https://github.com/Rnen/AdminToolbox/blob/master/.gitignore#L325-L332
```gitignore
# Smod & Stuff
Smod2.dll
Assembly-CSharp.dll
sourcecode.zip
Builds/
obj/
AdminToolbox/Checklist.txt
AdminToolbox/Checklist.txt
```

**Reply by ElectronicsArchiver** [reply]
So what besides the `following section` do you actually need?

**Reply by Rnen** [reply]
A lot of it, Temporary VS files should not be committed. 
I'll get a definite overview later when I'm home from work.
